requirements.txt

```diff
@@ -2,13 +2,10 @@ Flask==2.1.0
 flask_cors==3.0.9
 flask_profiler==1.8.1
 flask_restx==0.5.1
-werkzeug==2.1.2 # to fix bug https://github.com/python-restx/flask-restx/issues/460
-astropy==4.0.1.post1
```

Comment:
Are these not needed then ?

Reply:
werkzeug was repeated (it's still in the requirements) and astropy is no longer needed

Reply:
done